parsl/monitoring/radios.py

```diff
@@ -21,6 +24,37 @@ class MonitoringRadio(metaclass=ABCMeta):
     def send(self, message: object) -> None:
         pass
 
+class DateTimeEncoder(json.JSONEncoder):
+    def default(self, obj):
+        if isinstance(obj, (datetime.datetime, datetime.date, datetime.time)):
+            return obj.isoformat()
+        return super(DateTimeEncoder, self).default(obj)
+
+class DiasporaRadio(MonitoringRadio):
+    def __init__(self, monitoring_url: str, source_id: int, timeout: int = 10):
+        from diaspora_event_sdk import KafkaProducer
```

Comment:
Where does this package come from? If it's from a package that you can install from pypy using `pip install whatever`, then have a look in setup.py - you can add a new install option such as `pip install parsl[diaspora]` in there that will install the extra dependencies required here.